package.json

```diff
@@ -26,7 +26,7 @@
   },
   "peerDependencies": {
     "google-protobuf": "^3.13.0",
-    "typescript": "4.x.x"
+    "typescript": "4.x.x || 5.x.x"

```

Comment:
I'd suggest `>= 4.9.x` instead, to exactly match the runtime check.

Reply:
@drbild Good point - thanks. Updated.